packages/bun/test/integrations/bunserver.test.ts

```diff
@@ -1,67 +1,87 @@
-import { beforeAll, beforeEach, describe, expect, test } from 'bun:test';
+import { afterEach, beforeAll, beforeEach, describe, expect, test } from 'bun:test';
+import type { Span} from '@sentry/core';
 import { getDynamicSamplingContextFromSpan, setCurrentClient, spanIsSampled, spanToJSON } from '@sentry/core';
 
 import { BunClient } from '../../src/client';
 import { instrumentBunServe } from '../../src/integrations/bunserver';
 import { getDefaultBunClientOptions } from '../helpers';
 
-// Fun fact: Bun = 2 21 14 :)
-const DEFAULT_PORT = 22114;
-
 describe('Bun Serve Integration', () => {
   let client: BunClient;
+  // Fun fact: Bun = 2 21 14 :)
+  let port: number = 22114;
```

Comment:
We can't reuse the port on each start since `server.stop` happens in the background and tests end up using each other's servers.